Utils/github_workflow_scripts/trigger_contribution_build.py

```diff
@@ -40,7 +42,7 @@ def trigger_generic_webhook(options):
     body = {
         "name": "GenericWebhook_trigger_contribution_build",
         "raw_json": {"BaseBranch": base_branch, "PullRequestNumber": pr_number, "ContribBranch": contrib_branch,
-                     "ProjectID": GITLAB_PROJECT_ID},
+                     "ProjectID": GITLAB_PROJECT_ID, "ContribRepo": contrib_repo},
```

Comment:
if I understand correctly, this will allow defining the env variable in both builds, right? does it happen in gold? a bit hard to follow

Reply:
This request is for our Gold (`GenericWebHook`) and it will be mapped (using the `Contribution Build Incoming Mapper`) to a new incident field (`incident_contributionreponame`) and will eventually get to our Gitlab pipeline.